releasenotes/notes/trace-idle-timeout-20328713b855263d.yaml

```diff
@@ -0,0 +1,11 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+fixes:
+  - |
+    Fix idle timeout used to connect to datadog for trace
```

Comment:
```suggestion
    Fix idle timeout used to connect to Datadog for trace
```

Reply:
nit: can we be a bit more explicit here? We should mention what is the actual fix, even if it's a one liner.

Reply:
fixed